troi/musicbrainz/tests/test_related_artist_credits.py

```diff
@@ -1,68 +1,55 @@
-import json
 import os
 import unittest
 import unittest.mock
 
 import troi
-import troi.musicbrainz.recording_lookup
+import troi.musicbrainz.related_artist_credits
 
-request_json = [
-                    { "[recording_mbid]": "a96bf3b6-651d-49f4-9a89-eee27cecc18e" }, 
-                    { "[recording_mbid]": "ec5b8aa9-7483-4791-a185-1f599a0cdc35" }
-               ]
-
-return_json = [
+return_json = """[
```

Comment:
this is a string but others in `test_recording_lookup.py` and `test_area_random_recordings.py` use an object and json.dumps when setting the mock's return value

Reply:
Ah, yes. Fixed.